7_deploy_osh/roles/suse-osh-deploy/files/085-horizon.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Can we use component_install.yml rather than bringing this in when it's basically the same as what we have upstreaM?